src/utils/DateUtils.ts

```diff
@@ -1,24 +1,125 @@
 import { Options } from '../misc/Options';
 import { Utils } from './Utils';
 import { l } from '../strings/Strings';
-import { TimeSpan } from './TimeSpanUtils';
-import * as Globalize from 'globalize';
 import * as _ from 'underscore';
-
+import * as moment from 'moment';
+/**
+ * The 'DateUtils' class renders dates and times using momentJS using the right culture languages and formats.
+ *
+ * This class can transform strings, numbers and date objects into a standardized ISO_8601 Date.
+ *
+ */
 export interface IDateToStringOptions {
```

Comment:
This description suggests that we should pass a boolean value to this option (because of the word "wether"). I think we should change it to:

> Contains a standard Date object that specifies the current date and time.